### ISSUE TITLE
Add docstring to the repo

### ISSUE THREAD

**Issue by lyie28** [lavague-ai/LaVague#158]

**Comment by lyie28** [comment]
I agree. I am off for the rest of the week so I won't be able to add any until next week now. If you want to have a go at any, feel free to create a PR - at the latest, I will review on Monday when I am back in the office.

**Comment by cgmd5283** [comment]
seems doable

**Comment by tharunarasu** [comment]
Hi @lyie28 @shubhamofbce   I would like to work on adding docstrings contribute to LaVague and loved the vision. If anyone can explain on the objective I can start actively contributing.

**Comment by shubhamofbce** [comment]
@tharunarasu The objective is very simple. We just need to add docstrings to each and every class and method. You can start with a specific module and then cover the whole repo. This will help future contributors to understand classes and methods easily.
